src/main/webapp/WEB-INF/jsp/submitFeedback.jsp

```diff
@@ -20,91 +20,145 @@
+<form:form action="${postUrl}" modelAttribute="submitFeedbackForm">
     <spring:bind path="prefs.*">
-    	<c:if test="${status.error}">
-    	<div id="${n}error-message" class="error-message portlet-msg-error portlet-msg error" role="alert" style="display:none">
-   	 	<p><form:errors path="feedback"/></p>
-		</div>
-	    </c:if>
-	</spring:bind>
-	<br>
-	<p id="${n}answer">
-      	<form:radiobutton id="yes" path="like" value="YES"/>
-      	<label class="portlet-form-field-label"><spring:message code="feedback.answer.yes"/></label>
-      	<form:radiobutton id="no" path="like" value="NO"/>
-      	<label class="portlet-form-field-label"><spring:message code="feedback.answer.no"/></label>
-      	<form:radiobutton id="maybe" path="like" value="MAYBE"/>
-      	<label class="portlet-form-field-label"><spring:message code="feedback.answer.maybe"/></label>
-    </p>
-
- 	<p> 		
- 		<label class="portlet-form-field-label"><spring:message code="feedback.form.suggestion"/></label>
- 		<textarea id="${n}feedback" path="feedback" name="feedback" rows="${feedbackRows}" style="width:${feedbackWidth}"></textarea> 		
- 	</p>
- 	<div id="${n}limit" style="margin-bottom: 12px;"></div>
-
- 	<p>
-        <form:checkbox path="anonymous" value="true"/>
-		<label class="portlet-form-field-label"><spring:message code="feedback.form.anonymous"/></label>
-	</p>
-
- 	<input id="${n}useragentstring" type="hidden" name="useragent"/>
- 	<input id="${n}feedbacktabname" type="hidden" name="tabname"/>
+        <c:if test="${status.error}">
+        <div id="${n}error-message" class="alert alert-danger" role="alert" style="display:none">
+            <p><form:errors path="feedback"/></p>
+        </div>
+        </c:if>
+    </spring:bind>
+
+    <div data-role="fieldcontain">
+    <fieldset data-role="controlgroup" id="${n}answer">
+        <div class="radio radio-success">
+          <input type="radio" id="yes" name="like" value="YES"/>
+          <label for="yes" class="portlet-form-field-label"><spring:message code="feedback.answer.yes"/></label>
+        </div>
+        <div class="radio radio-danger">
+          <input type="radio" id="no" name="like" value="NO"/>
+          <label for="no" class="portlet-form-field-label"><spring:message code="feedback.answer.no"/></label>
+        </div>
+        <div class="radio radio-warning">
+          <input type="radio" id="maybe" name="like" value="MAYBE"/>
+          <label for="maybe" class="portlet-form-field-label"><spring:message code="feedback.answer.maybe"/></label>
+        </div>
+    </fieldset>
+    </div>
+
+     <p>
+         <label class="portlet-form-field-label" for="${n}feedback"><spring:message code="feedback.form.suggestion"/></label>
+         <textarea id="${n}feedback" name="feedback" class="form-control" rows="${feedbackRows}" style="width:${feedbackWidth}"></textarea>
+
+     </p>
+     <div id="${n}limit" style="margin-bottom: 12px;"></div>
+
+     <!-- p>
```

Comment:
I'm guessing that is commented out because anonymous feedback is not used at your institution. Can we have this uncommented?